nodejs/week2/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/", async (request, response) => {
```

Comment:
I think you could consider 2 more use cases in your implementation:

- What if I want to view all the meals without any query parameters at all. Would that work now?
- This is extra if you want to practice further!!!  What if I want to use more than one query parameter, for instance have both title and maxPrice? Currently you are using response.send directly, so I will get results based on only the first parameter.  What could be done differently?